executor/show_stats.go

```diff
@@ -82,6 +82,9 @@ func (e *ShowExec) appendTableForStatsHistograms(dbName, tblName, partitionName
 		return
 	}
 	for _, col := range statsTbl.Columns {
+		if col.IsInvalid(nil, false) {
```

Comment:
Why not pass sc here?

Reply:
Do we need to mark column stats needed in `show stats_histograms`?

Reply:
how about adding a comment like "pass a nil StatementContext to avoid column stats being marked as needed"?

Reply:
Comment added, PTAL